subprojects/platform-jvm/src/test/groovy/org/gradle/api/java/archives/internal/DefaultManifestTest.groovy

```diff
@@ -126,40 +124,6 @@ class DefaultManifestTest extends Specification {
         gradleManifest.effectiveManifest.getAttributes() == [key1: 'value1', key2: 'value2', key4: 'value4'] + MANIFEST_VERSION_MAP
     }
 
-    def write() {
-        Map testMainAttributes = [key1: 'value1', key2: 'value2', key3: 'value3', key4: 'value4'] as LinkedHashMap
-        Map testSectionAttributes = [sectionkey1: 'sectionvalue1']
-        String testSection = 'section'
-
-        DefaultManifest gradleManifest = new DefaultManifest(fileResolver)
-        gradleManifest.from(new DefaultManifest(fileResolver).attributes(testMainAttributes))
-        gradleManifest.attributes(testSectionAttributes, testSection)
-
-        Manifest expectedManifest = new Manifest()
-        expectedManifest.addConfiguredAttribute(new Attribute('key1', 'value1'))
-        expectedManifest.addConfiguredAttribute(new Attribute('key2', 'value2'))
-        expectedManifest.addConfiguredAttribute(new Attribute('key3', 'value3'))
-        expectedManifest.addConfiguredAttribute(new Attribute('key4', 'value4'))
-        expectedManifest.addConfiguredAttribute(new Attribute('Manifest-Version', '1.0'))
-        Section section = new Section()
-        section.setName testSection
-        section.addConfiguredAttribute(new Attribute('sectionkey1', 'sectionvalue1'))
-        expectedManifest.addConfiguredSection(section)
-        def StringWriter stringWriter = new StringWriter()
-
-        when:
-        gradleManifest.writeTo(stringWriter)
```

Comment:
Does `writeTo(Object)` maybe handle the test case properly as well? If it doesn't we might want to had a test case that shows what happens if you pass in a `StringWriter`. The use case: a user simply didn't change to a non-deprecated method however now `writeTo(Object)` would be called instead of `writeTo(Writer)`.